processing/src/main/java/io/druid/segment/IndexMerger.java

```diff
@@ -399,6 +416,35 @@ public String apply(@Nullable String input)
     return makeIndexFiles(indexes, outDir, progress, mergedDimensions, mergedMetrics, rowMergerFn, indexSpec);
   }
 
+  public static File reprocess(final File inDir, final File outDir, final IndexSpec indexSpec) throws IOException
```

Comment:
Following convention of other methods in this class, yes. From a "do we need it now" viewpoint? no